torch/_jit_internal.py

```diff
@@ -231,6 +231,32 @@ def can_compile_class(cls):
     return all(has_code)
 
 
+def get_callable_argument_names(fn):
+    """
+    Gets names of all POSITIONAL_OR_KEYWORD arguments for callable `fn`.
+    Returns an empty list when other types of arguments are present.
+
+    This is used by `torch.jit.trace` to assign meaningful argument names to
+    traced functions and modules.
+    """
+    # inspect.signature may fail, give up in that case.
+    try:
+        callable_signature = inspect.signature(fn)
+    except Exception:
+        return []
+
+    argument_names = []
+    for name, param in callable_signature.parameters.items():
+        # All four other types of arguments do not map to individual values
+        # with a keyword as name.
+        if not param.kind == param.POSITIONAL_OR_KEYWORD:
+            return []
+
```

Comment:
NIT: unnessary newline?

Reply:
Yeah, not necessary at all, but keeps code a bit cleaner IMHO : )

Reply:
haha cool cool